estate_account/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id/id,group_id/id,perm_read,perm_write,perm_create,perm_unlink
+estate.access_estate_property,access_estate_property,estate.model_estate_property,base.group_user,1,1,1,1
```

Comment:
This access rule exists already in the estate module

Reply:
Doesn't each module have to own its own access rules?! Is it because we inherit from the estate module, then its access rules apply here as well?